lib/fusuma/plugin/filters/libinput_device_filter.rb

```diff
@@ -57,13 +58,18 @@ def initialize(name_patterns:)
 
           # remove cache for reloading new devices
           def reset
-            @all = nil
+            @all = @accept_any = nil
             Device.reset
           end
 
+          def filter_record(record)
+            @accept_any || all.any? { |device| record.to_s =~ /^[\s-]?#{device.id}\s/ }
+          end
+
           # @return [Array]
           def all
             @all ||= if @name_patterns.empty?
+              @accept_any = true
```

Comment:
It appears that the use of `@accept_any = true` is causing the filtering by `Device.available` to be ineffective.

Reply:
I see. Is this an issue? I assumed that the use of filtering would be "negative filtering", i.e. something like "don't make fusuma try to process my touchscreen events as gestures". It seems that if there's no `name_patterns` then there is no need to filter events? I also expected that events would always come from "available" devices, is that wrong?